pkg/gardenlet/controller/backupbucket/backup_bucket_actuator.go

```diff
@@ -117,6 +117,9 @@ func (a *actuator) Delete(ctx context.Context) error {
 // by the current progress of the Flow execution.
 func (a *actuator) reportBackupBucketProgress(ctx context.Context, stats *flow.Stats) {
 	if err := kutil.TryUpdateStatus(ctx, kretry.DefaultRetry, a.gardenClient, a.backupBucket, func() error {
+		if a.backupBucket.Status.LastOperation == nil {
+			return fmt.Errorf("last operation of BackupBucket %s/%s is unset", a.backupBucket.Namespace, a.backupBucket.Name)
```

Comment:
Hm, why is this an error? Can't you simply initialize it to allow it being set in the next lines?

Reply:
See above